EGE/Prog/Lang.py

```diff
@@ -10,27 +10,29 @@ class ops:
     logic = [ '&&', '||', '^', '=>', 'eq' ]
     bitwise = [ '&', '|' ]
     unary = [ '++%s', '--%s', '%s++', '%s--', '!', '+', '-' ]
-    prio_unary = [ f"`{op}" for op in ops.unary ]
+    prio_unary = [ f"`{op}" for op in unary ]
 
     between = { 'between': [ '&&', [ '<=', 2, 1 ], [ '<=', 1, 3 ] ] }
 
 class Lang:
     def __init__(self, params):
         # my $self = { %init };
-        self.make_priorities
+        self.html = params.html
+        self.prio = {}
+        self.make_priorities()
 
-    def to_html(s: str):
+    def to_html(self, s: str):
         return html.escape(s)
 
     def op_fmt(self, op):
-        fmt = self.translate_op[op] or op
+        fmt = self.translate_op([op]) or op
```

Comment:
I think here need remove [] in call translate_op

Reply:
was not found usages of this functions

Reply:
With ToriRori find usage of this functions. Need to write like this: `translate_op()[op]`